client/flutter/lib/pages/latest_numbers.dart

```diff
@@ -20,118 +20,167 @@ const name = TextStyle(
-                    child: PageButton(
-                      Color(0xFF3D8AC4),
-                      S.of(context).latestNumbersPageViewLiveData,
-                      () => _launchStatsDashboard(context),
-                      verticalPadding: 24,
-                      borderRadius: 36,
-                      mainAxisAlignment: MainAxisAlignment.center,
-                      crossAxisAlignment: CrossAxisAlignment.center,
-                    ),
-                  )
-                ]));
-              }),
+                  content: Text(
+                    globalStats['cases'] != null
+                        ? numFmt.format(globalStats['cases'])
+                        : '-',
+                    softWrap: true,
+                    style: globalStats['cases'] != null ? number : loadingStyle,
+                    textAlign: TextAlign.left,
+                  )),
+              StatCard(
+                background: LatestNumbersGraph(
+                  timeseries: globalStats['timeseries'],
+                  timeseriesKey: 'dailyDeaths',
+                ),
+                title: Text(S.of(context).latestNumbersPageGlobalDeaths,
+                    style: name),
+                content: Text(
+                  globalStats['deaths'] != null
+                      ? numFmt.format(globalStats['deaths'])
+                      : '-',
+                  softWrap: true,
+                  style: globalStats['deaths'] != null ? number : loadingStyle,
+                  textAlign: TextAlign.left,
+                ),
+              ),
+              Container(
+                height: 25,
+              ),
+              Text(
+                this.lastUpdated ?? S.of(context).latestNumbersPageUpdating,
+                style: TextStyle(color: Color(0xff26354E)),
+                textAlign: TextAlign.center,
+              ),
+              Container(
+                height: 10,
+              ),
+              Text(
+                globalStats['attribution'] != null
+                    ? 'Source: ${globalStats['attribution']}'
+                    : '',
+                style: TextStyle(color: Color(0xff26354E)),
+                textAlign: TextAlign.center,
+              ),
+              Padding(
+                padding: const EdgeInsets.only(
+                  top: 24,
+                  left: 24,
+                  right: 24,
+                ),
+                child: PageButton(
+                  Color(0xFF3D8AC4),
+                  S.of(context).latestNumbersPageViewLiveData,
+                  () => _launchStatsDashboard(context),
+                  verticalPadding: 24,
+                  borderRadius: 36,
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                ),
+              )
+            ]),
+          ),
         ]);
   }
+
+  void initStats() async {
+    setState(() {
+      this.globalStats = Map();
+    });
```

Comment:
If the intent is for this to always have a value it might be clearer to initialize the map when it is declared or in an explicitly non-async initialization method.  This is currently valid because the async method runs synchronously until the first `await` is encountered, but if some code were later moved before it then there could be a build call before it was initialized.

Reply:
This has to be here so it resets every time

Reply:
I see now.  Do we want the UI to revert to blank while it's reloading or to show the previous data until the new data is loaded?  I think what you are doing looks fine, just wasn't expecting it.  (Ignore my previous comment).

Reply:
I think it is clearer that it is loading if they see the data re-appear,
I can change if @davidkaneda disagrees